com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -435,14 +499,12 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                         // to get them before the SSAO pass.
 
                         int gbufferNormalIndex = m_DeferredLights.GBufferNormalSmoothnessIndex;
-                        m_DepthNormalPrepass.Setup(cameraTargetDescriptor, m_ActiveCameraDepthAttachment, m_GBufferHandles[(int)DeferredLights.GBufferHandles.NormalSmoothness]);
+                        m_DepthNormalPrepass.Setup(cameraTargetDescriptor, m_ActiveCameraAttachments.depth, m_GBufferHandles[(int)DeferredLights.GBufferHandles.NormalSmoothness]);
 
                         // Change the normal format to the one used by the gbuffer.
                         RenderTextureDescriptor normalDescriptor = m_DepthNormalPrepass.normalDescriptor;
                         normalDescriptor.graphicsFormat = m_DeferredLights.GetGBufferFormat(gbufferNormalIndex);
                         m_DepthNormalPrepass.normalDescriptor = normalDescriptor;
-                        // Depth is allocated by this renderer.
-                        m_DepthNormalPrepass.allocateDepth = false;
```

Comment:
?

Reply:
We can no longer allocate within a pass because this would change the reference in this class from null to an RTHandle.
Previously we would have the same `RenderTargetHandle` with an id that would `GetTemporaryRT` but otherwise wouldn't change the object.
Now It's trickier because you have times where would use the Handle before it is allocated, so I moved to allocate within `UniversalRenderer`.

Reply:
If you would try to use Handle before the resource is allocated it would also be a problem in GetTemporaryRT, unless the handle was something special like BuiltinRenderTextureType. We should probably discuss further this. Let's setup a call so we can go through the code together and look at some cases.